### PR TITLE
Improvements to download NAM files for tc-forced nowcasts.

### LINKED ISSUE #654: NAM data acquisition for tc-forced nowcasts

### PR DESCRIPTION
**Pull request by jasonfleming**
resolves #654

### CODE REVIEW

File: asgs_main.sh
Comment (wwlwpd) [inline asgs_main.sh:2180]: I think we need to prefer managing these as `case/esac` just to plan for growth.  Also, checking a _negative_ (e.g., `!= "namBlend")` is asking for trouble, let's make this more positive :) - and `case/esac`.

File: asgs_main.sh
Comment (wwlwpd) [inline asgs_main.sh:2119]: let's do this as a `case/esac`

**Review by wwlwpd — CHANGES_REQUESTED** [review]
see comments

File: asgs_main.sh
Comment (jasonfleming) [reply asgs_main.sh:2119]: Excellent, can do. Thank you.

**Comment by jasonfleming** [comment]
I replaced the if/thens with case/esacs along with logging and reliability improvements. Have not tested yet.

**Comment by wwlwpd** [comment]
Looks *a lot* better! I didn't see any logical or syntax issues with it, but let me know how you'd like to proceed with testing; and I can help. Otherwise, let me know that it's working as you expected and I'll approve it and merge it.

**Review by wwlwpd — APPROVED** [review]

**Comment by wwlwpd** [comment]
@jasonfleming feel free to merge this once you're convinced it's working as expected - looks good to me, but I'll defer to you

**Pull request merged**
